src/download/index.test.ts

```diff
@@ -115,108 +104,56 @@ describe("Download Command", () => {
   });
 
   setup()
-    .do(() => DownloadCommand.run(["--group-by", "CC API Family"]))
+    .do(() => DownloadCommand.run([]))
     .it("downloads APIs", () => {
       expect("apis").to.be.a.directory();
-      expect("apis/api-config.json")
+      expect("apis/shopper-customers/.metadata.json")
         .to.be.a.file()
-        .with.content(`${API_CONFIG_JSON}\n`);
+        .with.content(API_CONFIG_JSON + "\n");
```

Comment:
Curious why you changed this.

Reply:
I honestly have no idea.  And no recollection of changing this.  Would fix it in the future though as it doesn't affect anything